src/modules/intelligence/application/event_handlers/analysis_completed_handler.py

```
@@ -33,103 +39,113 @@ def __init__(
-                    ))
-
-                # ── Tag & group translation ───────────────────────────────────
-                try:
-                    self._translate_tags_uc.translate_tags(lang, limit=50)
-                except Exception as e:
-                    lang_span.record_exception(e)
-                    _logger.error("auto_tag_translation_error", language=lang, error=str(e))
+                    # ── Tag & group translation ───────────────────────────────────
+                    try:
+                        await self._translate_tags_uc.translate_tags(lang, limit=50)
+                    except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)